app/forms/enterprise_fees_bulk_update.rb

```diff
@@ -1,75 +1,74 @@
 # frozen_string_literal: true
 
-module Forms
-  class EnterpriseFeesBulkUpdate
-    include ActiveModel::Model
+class EnterpriseFeesBulkUpdate
+  include ActiveModel::Model
 
-    validate :check_enterprise_fee_input
-    validate :check_calculators_compatibility_with_taxes
+  attr_reader :errors
 
-    def initialize(params)
-      @errors = ActiveModel::Errors.new self
-      @params = params
-    end
-
-    def save
-      return false unless valid?
+  validate :check_enterprise_fee_input
+  validate :check_calculators_compatibility_with_taxes
 
-      @enterprise_fee_set = Sets::EnterpriseFeeSet.new(enterprise_fee_bulk_params)
-      unless @enterprise_fee_set.save
-        @enterprise_fee_set.errors.each do |attribute, message|
-          @errors.add(attribute, message)
-        end
-        return false
-      end
+  def initialize(params)
+    @errors = ActiveModel::Errors.new self
+    @params = params
+  end
 
-      true
+  def save
+    return false unless valid?
 
+    @enterprise_fee_set = Sets::EnterpriseFeeSet.new(enterprise_fee_bulk_params)
+    unless @enterprise_fee_set.save
+      @enterprise_fee_set.errors.each do |attribute, message|
+        @errors.add(attribute, message)
+      end
+      return false
```

Comment:
Nice ! 
Although your test do not cover this. One way to do that without having to step up complicated data to make `@enterprise_fee_set.save` would be to mock `@enterprise_fee_set`  ie  :
```ruby
# set up a mock instance with a method save returning false
enterprise_fee_set = instance_double(Sets::EnterpriseFeeSet, save: false)

# Setting some errors
test_errors = ActiveModel::Errors.new(enterprise_fee_set) 
test_errors.add(:base, "some test error")
# Stubbing enterprise_fee_set errors method to return the test erros we just created
allow(enterprise_fee_set).to receive(:errors).and_return(test_errors)

# Stub new, so it returns the mock we just created
allow(Sets::EnterpriseFeeSet).to receive(:new).and_return(enterprise_fee_set)
```
Then you can test `EnterpriseFeesBulkUpdate.errors` includes errors in `test_errors`.
Hopefully that's enough to get you started, I have not tried this code so it might need some fixing. Ping me is anything is not clear.

Reply:
> Hopefully that's enough to get you started, I have not tried this code so it might need some fixing. Ping me is anything is not clear.

I added an additional test for that. Would you like me to break out the name change/file into a separate commit and force-push it?

Reply:
That would be great, thanks !

Reply:
I **think** I split the commits correctly.

Reply:
@macanudo527 sorry to be difficult, you did split the commit but now the commit messages don't match the commits. Could you fix that ? Let me know if you need some hep with that. Thanks!

Reply:
I can change the messages, but they appear to be okay to me. Do none of them match? I mean "Move and Rename Form" seems to cover moving the form. Did something not get staged maybe?

Reply:
This tells me "[Move and Rename BulkUpdateForm to Forms Folder](https://github.com/openfoodfoundation/openfoodnetwork/pull/11150/commits/dfae4892aef283afce2c0d079332dfd77c1905ab)" but when I look at the code changes it shows spec changes. Same for this one https://github.com/openfoodfoundation/openfoodnetwork/pull/11150/commits/da43de1bb06e996290426b752b74885b391b81bc, the commit message doesn't match the code changes.

Reply:
Sorry about that. I see what I did. Should be okay now.